NachoSpigot-Server/src/main/java/net/minecraft/server/TileEntitySign.java

```diff
@@ -9,6 +9,7 @@
     public boolean isEditable = true;
     private EntityHuman h;
     private final CommandObjectiveExecutor i = new CommandObjectiveExecutor();
+    public java.util.UUID signEditor;
```

Comment:
```suggestion
    //private EntityHuman h; // Paper
    private final CommandObjectiveExecutor i = new CommandObjectiveExecutor();
    public java.util.UUID signEditor; // Paper
```

Reply:
Im on my mobile but is the entityhuman variable in generel useless?

Reply:
Basically, since all calls to it was removed